src/main/resources/assets/admin/common/js/ui/treegrid/TreeGrid.ts

```diff
@@ -1698,7 +1698,7 @@ export class TreeGrid<DATA extends IDentifiable>
         this.setActive(true);
     }
 
-    private highlightRowByNode(node: TreeNode<DATA>) {
+    private highlightRowByNode(node: TreeNode<DATA>, notify: boolean = true) {
```

Comment:
We usually do it the other way around: call such a parameter `silent` and make it false by default. You will find this all over the code.